### PR TITLE
Feature/pastelstatistics average-block-size chart

### PR DESCRIPTION
**Pull request by Dinisueldanov**
### Summary
- added average block size chart.
- added tests.

### Screenshots
main-averageblock
averageblock

### CODE REVIEW

File: src/features/pastelStatistics/components/chart/EChartsLineChart.tsx [hunk 1]
```diff
@@ -62,13 +72,26 @@ export const EChartsLineChart = (props: TLineChartProps): JSX.Element => {
         })
         setCsvData(data)
       }
+    } else if (dataY1?.length && dataY2?.length) {
+      if (dataX) {
+        const data: Data = []
+        dataY1.map((o, index) => {
```
Comment (ngvtuan) [inline src/features/pastelStatistics/components/chart/EChartsLineChart.tsx:78]: `o` isn't make sense. Can we change the name?

File: src/features/pastelStatistics/utils/ChartOptions.ts [hunk 2]
```diff
@@ -127,6 +127,218 @@ export function getThemeInitOption(args: TThemeInitOption): EChartsOption {
...
+            const val = Number(value).toFixed(2)
+            return `${val} k`
+          },
+        },
+      },
+      series: {
+        type: 'line',
+        sampling: 'lttb',
+        lineStyle: {
+          color: 'rgb(23,105,135)',
```
Comment (ngvtuan) [inline src/features/pastelStatistics/utils/ChartOptions.ts:274]: Can we use `hex` for the color as in other sections?

**Review by ngvtuan — COMMENTED** [review]
Please remove all code isn't belong to this PR (`MEMPOOLSIZEOVERTIME ` and `NETTOTALSOVERTIME ` charts)

UI issue 1
- Can we change the date format like other charts? 
- Can we update the text to `Period `
- Can we update the Y-axis range? Can we use `0, 1, 2, 3 ...` instead of `1.5, 1.6, 1.8, 2 ... `? 

Add index.ts for each component, please

**Review by ngvtuan — COMMENTED** [review]
This PR is ok to merge. But as discussed in the group, we need to remove  unused code 
unuse code

**Review by ngvtuan — APPROVED** [review]

**Review by westalert — APPROVED** [review]

**Pull request merged**